src/applications/check-in/travel-claim/pages/travel-review/index.jsx

```diff
@@ -113,9 +117,9 @@ const TravelQuestion = props => {
       bodyText={bodyText}
       pageType="travel-review"
       router={router}
-      yesButtonText={t('agree-to-these-terms')}
+      yesButtonText={t('file-claim')}
       yesFunction={validation}
-      noButtonText={t('file-later')}
+      noButtonText={t('back')}
       noFunction={fileLater}
```

Comment:
the noFunctions here should go back to the last page, we don't need to record the answers in this app either.

Reply:
also same as address page, we can now add the address here since it is in mock data.

Reply:
Another content change is that the Beneficiary travel agreement has been moved from the additional info component to its own page in figma.